platform/ios/src/MGLScaleBar.mm

```diff
@@ -185,7 +185,7 @@ - (void)commonInit {
 #pragma mark - Dimensions
 
 - (CGSize)intrinsicContentSize {
-    return CGSizeMake(self.actualWidth + self.lastLabelWidth/2, 16);
+    return CGSizeMake(ceil(self.actualWidth + self.lastLabelWidth/2), 16);
```

Comment:
`MGLPointRounded` may be a better choice here than `ceil`, since it’s meant to account for pixel scale.